pommerman/forward_model.py

```diff
@@ -270,7 +270,8 @@ def crossing(current, desired):
                     change = True
 
         # Handle kicks.
-        bombs_kicked_by = dict()
+        agent_indexed_by_kicked_bomb = dict()
```

Comment:
s/dict()/{} to align with the rest of the file.
while you're at it, if you could change "crossings = dict()" as well, that would be helpful